JetStreamCompose/jetstream/src/main/java/com/google/jetstream/data/util/StringConstants.kt

```diff
@@ -124,7 +124,6 @@ object StringConstants {
         const val VideoPlayerControlClosedCaptionsButton = "Playlist Button"
         const val VideoPlayerControlSettingsButton = "Playlist Button"
         const val VideoPlayerControlPlayPauseButton = "Playlist Button"
-        const val SampleVideoUrl =
-            "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4"
```

Comment:
This video was approved by the legal team to be used within the JetStream app and we also have its license mentioned in the license.md file: https://github.com/android/tv-samples/blob/main/JetStreamCompose/LICENSES.md

We might need similar approval and license update for the new Elephant's video who's link was added in the `movies.json` file.

Reply:
Ah, we can simply use the Big Buck Bunny video instead. I don't have a strong preference for one or the other. Made the change, so I'll resolve this comment!